### PR TITLE
Update/webshoppingagg webapp builder

### LINKED ISSUE #2095: WebShoppingAgg: Migrate to WebApplicationBuilder

### PR DESCRIPTION
**Pull request by erjain**
Migrate webshoppingagg to webapp builder
Remove Startup.cs

### CODE REVIEW

**Comment by erjain** [comment]
Adding @ReubenBond to review.

File: src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs [hunk 1]
```diff
@@ -1,24 +1,225 @@
-await ﻿BuildWebHost(args).RunAsync();
+﻿var appName = "Ordering.API";
+var builder = WebApplication.CreateBuilder(args);
+builder.WebHost.ConfigureAppConfiguration(cb =>
+{
+    var sources = cb.Sources;
+    sources.Insert(3, new Microsoft.Extensions.Configuration.Json.JsonConfigurationSource()
```
Comment (ReubenBond) [inline src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs]: This is extremely weird. Hard no on this approach

File: src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs [hunk 2]
```diff
@@ -1,24 +1,225 @@
...
+{
+    var seqServerUrl = configuration["Serilog:SeqServerUrl"];
+    var logstashUrl = configuration["Serilog:LogstashgUrl"];
+    return new LoggerConfiguration()
+        .MinimumLevel.Verbose()
+        .Enrich.WithProperty("ApplicationContext", Program.AppName)
+        .Enrich.FromLogContext()
+        .WriteTo.Console()
+        //.WriteTo.Seq(string.IsNullOrWhiteSpace(seqServerUrl) ? "http://seq" : seqServerUrl)
+        //.WriteTo.Http(string.IsNullOrWhiteSpace(logstashUrl) ? "http://logstash:8080" : logstashUrl, null)
```
Comment (ReubenBond) [inline src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs]: We should not commit commented-out code. Just delete it.

Comment (erjain) [reply src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs]: Sure Reuben

File: src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs (hunk 1 above)
Comment (erjain) [reply src/ApiGateways/Web.Bff.Shopping/aggregator/Program.cs]: May be we can remove this section?

**Review by ReubenBond — APPROVED** [review]

**Pull request merged**